src/confd/src/ietf-interfaces.h

```diff
@@ -63,6 +75,11 @@ int bridge_gen_ports(struct dagger *net, struct lyd_node *dif, struct lyd_node *
 int netdag_gen_bridge(sr_session_ctx_t *session, struct dagger *net, struct lyd_node *dif,
 		      struct lyd_node *cif, FILE *ip, int add);
 
+/* infix-if-lag.c */
+int lag_gen_ports(struct dagger *net, struct lyd_node *dif, struct lyd_node *cif, FILE *ip);
+int netdag_gen_lag(sr_session_ctx_t *session, struct dagger *net, struct lyd_node *dif,
+		   struct lyd_node *cif, FILE *ip, int add);
+
```

Comment:
To be consistent with the bridge code (after the refactor), I suggest `lag_gen()` and `lag_port_gen()`

Reply:
Thanks, yep!